app/assets/app.js

```diff
@@ -0,0 +1,3 @@
+//= require js/vendor/jquery-2.0.3.js
```

Comment:
sprockets wants these entrypoint files to require the other css and js files
require jquery first, since everything else depends on it